lib/files.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs')
```

Comment:
Let's rename the file to something a bit more meaningful. Something like `files-utils` fits better to the purpose of the file.